.travis.yml

```diff
@@ -1,12 +1,13 @@
 sudo: false
 language: node_js
+cache:
+  npm: false
 notifications:
   email: false
 node_js:
   - '10'
   - '8'
   - '6'
-  - '4'
```

Comment:
dropping support for node 4 is not acceptable, i'm afraid.

Reply:
Ok, that is yargs https://github.com/yargs/yargs/blob/master/package.json#L77

Reply:
yargs v11 supports node 4.

Reply:
But unfortunately does not fix "mem"

Reply:
then this isn't a fixable non-problem, and you'll just have to ignore the warning.

Reply:
Ah well, thanks.